os/booting-on-cloudca.md

```diff
@@ -0,0 +1,254 @@
+# Running CoreOS on cloud.ca
+
+[cloud.ca](http://www.cloud.ca) is a regional IaaS designed to help meet
+regulatory or security requirements by storing data in Canada. CoreOS is a
+supported operating system for VMs on the cloud.ca infrastructure.
+
+
+## Choosing a Channel
+
+CoreOS is designed to be [updated automatically][update-docs], with three
+channels of different update schedules: Stable, Beta, and Alpha. You can
+[disable this feature][reboot-docs], although we don't recommend it. Read the
+[release notes][release-notes] for specific features and bug fixes.
+
+The cloud.ca CoreOS image is built from the official stable release CloudStack
+image.
+
+<div id="cca-images">
+  <ul class="nav nav-tabs">
+    <li class="active"><a href="#stable" data-toggle="tab">Stable Channel</a></li>
+    <li><a href="#beta" data-toggle="tab">Beta Channel</a></li>
+    <li><a href="#alpha" data-toggle="tab">Alpha Channel</a></li>
+  </ul>
+  <div class="tab-content coreos-docs-image-table">
+    <div class="tab-pane" id="alpha">
+      <div class="channel-info">
+        <p>The Alpha channel closely tracks master and is released frequently.
+        The newest versions of <a href="{{site.baseurl}}/using-coreos/docker">Docker</a>,
+        <a href="{{site.baseurl}}/using-coreos/etcd">etcd</a> and
+        <a href="{{site.baseurl}}/using-coreos/clustering">fleet</a> will be
+        available for testing. To use the Alpha channel on cloud.ca, select the
+        "CoreOS Stable" template and update the cloud-config to
+        [change the channel to alpha][switching-channels]:
+        </p>
+      </div>
+```yaml
```

Comment:
Because this is in a chuck of HTML, I think you'll need to use `<pre>` tag here because the markdown renderer won't touch this.